From bb67d22352ffab6fcbaaa9aa8218f49a0d5d1028 Mon Sep 17 00:00:00 2001
From: Matt Ranostay <mranostay@ti.com>
Date: Sat, 12 Feb 2022 02:39:28 +0200
Subject: [PATCH 11/14] arm64: dts: expanded k3-am654 memory region for
 jailhouse overlay

Memory window expanded to allow an additional IVSHMEM device to be
mapped between inmates and root cell.

Signed-off-by: Matt Ranostay <mranostay@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am654-base-board-jailhouse.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am654-base-board-jailhouse.dts b/arch/arm64/boot/dts/ti/k3-am654-base-board-jailhouse.dts
index 762bce28fb9e..2da45b18ae68 100644
--- a/arch/arm64/boot/dts/ti/k3-am654-base-board-jailhouse.dts
+++ b/arch/arm64/boot/dts/ti/k3-am654-base-board-jailhouse.dts
@@ -14,8 +14,8 @@ &reserved_memory {
 	 * Reserve for Jailhouse hypervisor firmware and inmates.
 	 * Must be in sync with jailhouse cell configurations.
 	 */
-	hyp_mem: jailhouse@8dfb00000 {
-		reg = <0x8 0xdfb00000 0x0 0x20500000>;
+	hyp_mem: jailhouse@8dfa00000 {
+		reg = <0x8 0xdfa00000 0x0 0x20600000>;
 		alignment = <0x1000>;
 		no-map;
 	};
-- 
2.30.2

